README.md

```diff
@@ -63,54 +63,32 @@ Shipwright supports any tool that can build container images in Kubernetes clust
       --docker-email=<your_email>
   ```
 
-* Create a *Build* object, replacing `<REGISTRY_ORG>` with the registry username your `push-secret` secret have access to:
+- Create and submit your *BuildRun* object, replacing `<REGISTRY_ORG>` with the registry username your `push-secret` secret have access to:
```

Comment:
Yeah, I was thinking about this one, because there were some linter issues I wanted to resolve anyway. So my rational was: The example in the README is a quick try it out kind of example. So it would be nice if it would be as simple as possible. What do you think?

Reply:
It should be simple, I agree. But, it also is the first sample that our users might see. If we show the BuildRun-only use case, then I think we must at least make clear that this is not the only possibility but maybe used as the simplest option to verify the installation.

Reply:
Plus one on not changing this. I prefer to introduce a sentence to highlight on this other feature or to extend into a new tutorial using the embedded `BuildSpec`. The design of SHP is based on two core resources, this is what we should pitch on our landing document, as using a Build + BuildRun have a larger scope in terms of use cases than the one-off Build.

Reply:
Removed.